package.json

```diff
@@ -74,6 +72,8 @@
     "@fabric-ds/icons": "0.6.7",
     "@open-wc/testing": "3.2.0",
     "@warp-ds/core": "1.0.0",
+    "@warp-ds/uno": "^1.0.0-alpha.49",
+    "@warp-ds/component-classes": "^1.0.0-alpha.116",
```

Comment:
these are moved from devDependenices to dependencies, is it intentional? :)

Reply:
Yes, the idea was to prevent [renovate bot](https://docs.renovatebot.com/) from auto-merging patch-version updates in case we implement this tool in the same way as it was done in [Fabric](https://github.com/fabric-ds/elements/blob/fcf8315a8710ea7488033495fa7b73f2fd427d7d/.github/renovate.json#L4). We were thinking that changes from @warp-ds packages might require some testing before version updates are merged in component packages.

On a side note, I was also thinking that eventually these dependencies could be treated as peer-dependencies so that we let users know which future releases are compatible with which versions of other packages, but I haven't yet looked too much into this subject (just found [this article](https://code-trotter.com/web/dependencies-vs-devdependencies-vs-peerdependencies/)).

Reply:
Actually, it seems the peer-dependency warning works fine without us doing anything :D Example from installing the newest Warp React in @warp-ds/tech-docs:
<img width="624" alt="Screenshot 2023-07-10 at 13 42 30" src="https://github.com/warp-ds/elements/assets/41303231/bacf3cca-313b-4db7-90dd-247cbf352d2c">